src/js/vic-v2/components/PhotoField.jsx

```diff
@@ -388,55 +413,81 @@ export default class PhotoField extends React.Component {
 
   render() {
     const file = this.props.formData || {};
-    const errorMessage = file.errorMessage || this.state.errorMessage;
+    const { isCropping } = this.state;
+    const hasFile = !!file.confirmationCode;
+    const errorMessage = file.errorMessage;
+    const screenReaderError = this.screenReaderPath && !!errorMessage;
+    const label = this.props.uiSchema['ui:title'];
     const smallScreen = isSmallScreen(this.state.windowWidth);
     const moveControlClass = ['cropper-control', 'cropper-control-label-container', 'va-button-link'];
     const fileTypes = this.props.uiSchema['ui:options'].fileTypes;
-    const isDone = !!file.confirmationCode;
     const progressBarContainerClass = classNames('schemaform-file-uploading', 'progress-bar-container');
+
+    let fieldView;
```

Comment:
I'm hoping this is a clearer way of understanding the different visual states in the field. Let me know if it isn't and I can adjust it.